addons/editor-messages/userscript.js

```diff
@@ -1,7 +1,6 @@
-export default async function ({ addon, console, msg }) {
+export default async function ({ addon, console }) {
   const messages = document.createElement("a");
   messages.href = "/messages/";
-  messages.title = msg("messages");
```

Comment:
Could this be re-added? It was the tooltip that appears when hovering over the icon.